### PR TITLE
 replace acr image with prometheus

### PR DESCRIPTION
**Pull request by yanjiaxin534**

### CODE REVIEW

**Review by msftcoderdjw — APPROVED** [review]

File: api/pkg/apis/v1alpha1/providers/target/k8s/k8s_test.go [hunk 1]
```diff
@@ -187,13 +187,13 @@ func TestDeploymentToComponents(t *testing.T) {
 	assert.Equal(t, 2, len(components))
 	assert.Equal(t, "evs", components[0].Name)
 	assert.Equal(t, "rocket", components[1].Name)
-	assert.Equal(t, "evaamscontreg.azurecr.io/evsclient:latest", components[0].Properties[model.ContainerImage])
+	assert.Equal(t, "prom/prometheus", components[0].Properties[model.ContainerImage])
 	assert.Equal(t, "evaamscontreg.azurecr.io/rocket:detection", components[1].Properties[model.ContainerImage])
```
Comment (lirenjie95) [inline api/pkg/apis/v1alpha1/providers/target/k8s/k8s_test.go:191]: Please also replace this image `evaamscontreg.azurecr.io/rocket:detection`

File: api/pkg/apis/v1alpha1/providers/target/k8s/k8s_test.go [hunk 2]
```diff
@@ -232,7 +232,7 @@ func TestComponentsToDeploymentFull(t *testing.T) {
 	assert.Equal(t, "name", d.Spec.Selector.MatchLabels["app"])
 	assert.Equal(t, "evs", d.Spec.Template.Spec.Containers[0].Name)
 	assert.Equal(t, "rocket", d.Spec.Template.Spec.Containers[1].Name)
-	assert.Equal(t, "evaamscontreg.azurecr.io/evsclient:latest", d.Spec.Template.Spec.Containers[0].Image)
+	assert.Equal(t, "prom/prometheus", d.Spec.Template.Spec.Containers[0].Image)
 	assert.Equal(t, "evaamscontreg.azurecr.io/rocket:detection", d.Spec.Template.Spec.Containers[1].Image)
```
Comment (lirenjie95) [inline api/pkg/apis/v1alpha1/providers/target/k8s/k8s_test.go:236]: same as above

File: api/pkg/apis/v1alpha1/providers/target/k8s/k8s_test.go (hunk 1 above)
Comment (yanjiaxin534) [reply api/pkg/apis/v1alpha1/providers/target/k8s/k8s_test.go]: Thanks， I will replace it in next version

Comment (lirenjie95) [reply api/pkg/apis/v1alpha1/providers/target/k8s/k8s_test.go]: I'm also wondering about `components[0].Name` or related fields. Should we also change them?

File: api/pkg/apis/v1alpha1/providers/target/k8s/k8s_test.go [hunk 3]
```diff
@@ -154,7 +154,7 @@ func TestDeploymentToComponents(t *testing.T) {
 						},
 						{
 							Name:            "rocket",
-							Image:           "evaamscontreg.azurecr.io/rocket:detection",
+							Image:           "prom/prometheus",
 							ImagePullPolicy: "Always",
 							Args:            []string{"pipeline=3", "line=https://aka.ms/lineeast960", "cat=car person"},
```
Comment (msftcoderdjw) [inline api/pkg/apis/v1alpha1/providers/target/k8s/k8s_test.go:159]: prom/prometheus doesn't support cmdline args like pipeline=xxxx line=xxxx, let's find two different images and better find one which needs command-line arguments.

**Review by msftcoderdjw — CHANGES_REQUESTED** [review]

**Review by lirenjie95 — APPROVED** [review]

**Review by msftcoderdjw — CHANGES_REQUESTED** [review]

**Review by msftcoderdjw — APPROVED** [review]

**Pull request merged**